app/controllers/v0/messages_controller.rb

```diff
@@ -98,7 +98,7 @@ def move
     private
 
     def message_params
-      @message_params ||= params.require(:message).permit(:category, :body, :recipient_id, :subject)
+      @message_params ||= params.require(:message).permit(:id, :draft_id, :category, :body, :recipient_id, :subject)
```

Comment:
@mphprogrammer might this be a good time to just force it to be draft_id? I am not sure I like having :id in strong parameters, even though this isn't active record and there is no mass assignment risk, it just seems really wrong. 

I'm open to arguments in favor, but maybe we just lock it in as draft_id.